src/test/java/seedu/address/logic/parser/DeleteReminderCommandParserTest.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REMINDER_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REMINDER_DESCRIPTION;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.DeleteReminderCommand;
+import seedu.address.model.date.ReminderDate;
+import seedu.address.model.reminder.ReminderDescription;
+
+public class DeleteReminderCommandParserTest {
```

Comment:
Can consider changing deleteReminder to using 2 indexes, one for person, one for reminder, as mentioned before. You can take a look at delete contacted info for reference